### DISCUSSION TITLE
split_video_ffmpeg error

### DISCUSSION THREAD

**Discussion by lsdlh** [Breakthrough/PySceneDetect#362] [General]
split_video_ffmpeg error
Opencv version 4.8.1.78
[mp4 @00000156386b7380] Could not find tag for codec h264 in stream #1, codec not currently supported in container
[out #0/mp4 @00000156386b7240] Could not write header (incorrect codec parameters?): Invalid argument
[Vost #0:0/libx264 @00000156386b8040] Error initializing output stream: 
Conversion failed!
Error splitting video (ffmpeg returned 1).

**Comment by Breakthrough** [comment]
Are you overriding any of the parameters passed to the function?  Depending on the format of your input, you might have to override them to handle specific use cases.  You can see the defaults by running `scenedetect help split-video`.
